SpinCore_pp/field_control.py

```diff
@@ -0,0 +1,23 @@
+from Instruments.XEPR_eth import xepr
+import pyspecdata as psd
+
+logger = psd.init_logging(level="info")
+
+def set_field(config_dict):
+    """Ensures the field is appropriate before setting the desired field."""
+    input(
+        "I'm assuming that you've tuned your probe to %f since that's what's in your .ini file. Hit enter if this is true"
+        % config_dict["carrierFreq_MHz"]
+    )
```

Comment:
This is not part of setting the field, and should go in the pulse sequence... The point of this is to make sure that the probe is tuned to the frequency you're going to be sending high power pulses at